tests/test_meshtastic_utils_message_edge.py

```
@@ -10,23 +10,31 @@
+import asyncio
 import contextlib
 import inspect
 import threading
+import time
 import unittest
 from collections.abc import Callable
 from concurrent.futures import Future
-from typing import Any
-from unittest.mock import MagicMock, Mock, patch
+from concurrent.futures import TimeoutError as ConcurrentTimeoutError
+from typing import Any, Callable, NoReturn
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)